{{cookiecutter.project_shortname}}/docs/license.rst

```diff
@@ -2,6 +2,7 @@ License
 =======
 
 .. include:: ../LICENSE
+   :literal:
```

Comment:
Just a suggestion, according to the documentation.

`.. literalinclude:: ../LICENSE`